ssh/src/main/java/ch/cyberduck/core/sftp/openssh/config/transport/OpenSshConfig.java

```diff
@@ -126,22 +126,21 @@ public Host lookup(final String hostName) {
     }
 
     public Map<String, Host> refresh() {
-        final long mtime = configuration.attributes().getModificationDate();
```

Comment:
`IOException` is already catched in `LocalAttributes` and ignored. This will then return `-1`. This implies the exception is not correctly mapped in IKVM.